### PR TITLE
merge Cassandra branch with master, moved old to its own branch

### PR DESCRIPTION
**Pull request by 3-Hz**

### CODE REVIEW

File: client/index.jsx [hunk 1]
```diff
@@ -16,9 +16,28 @@ class ReviewComponent extends React.Component {
   }
 
   componentDidMount() {
```
Comment (javalzar1) [inline client/index.jsx:18]: nice on updating the client to handle your backend changes.

File: server/db/seed.js [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  let tenth = (prog / limit * 10);
+  let progBar = ('[' + '='.repeat(tenth * 2) + '>'.repeat(tenth < 10)).padEnd(21) + ']';
+  process.stdout.write(`BatchT: ${batchtime}, Progress: ${(tenth * 10).toFixed(0)}%`.padEnd(20) + progBar);
+}
+
+const endProgress = () => {
+  resetCrg();
+  process.stdout.write(`Progress: 100%`.padEnd(20) + `[${'='.repeat(20)}]\n`);
+}
+
```
Comment (javier-zarate) [inline server/db/seed.js:34]: cool progress bar functions

**Review by javier-zarate — APPROVED** [review]
Everything looks good 👍🏻

File: client/index.jsx [hunk 3]
```diff
@@ -16,9 +16,28 @@ class ReviewComponent extends React.Component {
...
+          product: review.productid,
+          metrics: {
+              difficulty: review.difficulty,
+              value: review.value,
+              quality: review.quality,
+              appearance: review.appearance,
+              works: review.works
+          }
+        }));
+        //console.log(reviews[~~(Math.random() * reviews.length)]);
```
Comment (javier-zarate) [inline client/index.jsx:40]: nice on updating the client to handle your backend changes.

**Review by javier-zarate — APPROVED** [review]

**Pull request merged**